content/languages/sql/sql-writting-tests.md

```diff
@@ -0,0 +1,81 @@
+---
+description: ""
+kind: tutorial
+languages: [sql]
+topic: tests
+tags:
+  - testing
```

Comment:
```suggestion
  - testing
sidebar: "language:sql"
prev: /languages/sql/
```

`sidebar: "language:id"` will show all the pages related to the language.

Reply:
damn it, I didn't see this message before. 

Isn't there a way to get notifications for the reviews? (maybe it's sent by email? Then I'd have to change the one used for github... :/ )

Q: should we have a list of "topics" as we do for "kinds" or so? Because we might end up with `test/tests/testing` or so, at some point.